### ISSUE TITLE
Create a local WordPress website in Windows ran into problems

### ISSUE THREAD

**Issue by mysti9uemirage** [mysti9uemirage/PersonalRepo#2]
Trying to login at wp-admin will say "You must log in to access the admin area."
Trying to login at wp-login.php page got redirected to a 404 NOT FOUND error message

**Comment by mysti9uemirage** [comment]
https://managewp.com/how-to-create-a-local-copy-of-a-live-wordpress-site (has link to create fresh website)

===

**Might get 404 error when try to go to wp-login.php**. Follow this link to resolve (pasted relevant content here - from wpa):

1. In the "wp-content" folder I changed the name of the "plugins" folder to "plugins.off" (whichever name would be fine)
2. I logged at "[my-site-domain]/wp-login.php" but this time without to be redirected, so that I was able to enter the dashboard (I saw that it had error in mu-plugins folder, so I renamed it to off also)
3. I renamed back to "plugins" the "plugin.off" folder name (see step 1)
4. Within "Dashboard > Plugin" I reactivated all the plugins that were active before I experienced the issue of being redirected at login, all but one plugin which I think could be the culprit of the problem.
5. Logged-out and relogged-in, just to verify that the redirection disappeared.

**In short**: Plugins might cause problems... Turn them off to troubleshoot.
https://wordpress.org/support/topic/wp-loginphp-redirects-to-404-not-found

**Comment by mysti9uemirage** [comment]
**Sometimes, when login to wp-admin, got redirected to 127.0.0.1 instead.**

To resolve: https://codex.wordpress.org/Changing_The_Site_URL
define('WP_HOME','http://localhost/kaeinllc');
define('WP_SITEURL','http://localhost/kaeinllc');
define('RELOCATE',true); 
// Important! Leaving the RELOCATE constant in your wp-config.php file is insecure, as it allows an attacker to change your site URL to anything they want in some configurations. Always remove the RELOCATE line from wp-config.php after you're done.

Should also edit values in **wp_options table**.
